### ISSUE TITLE
Is there some way to disable and then enable? (jQuery)

### ISSUE THREAD

**Issue by merzikain** [noraesae/perfect-scrollbar#452]
I need to disable perfect-scrollbar when the browser is smaller than 1280px wide then re-enable it if the window is scaled to 1280px or larger.

I am using $('.perscroll').perfectScrollbar({suppressScrollX:true}); to initialize the plugin and if the window is smaller than 1280px I use this to "disable" it:
$('.perscroll').perfectScrollbar('destroy');

I then setup an event handler for $(window).on('resize') to check the window width and do whichever of those two actions I need.

The problem is that once it's destroyed I can't get it back. What am I doing wrong?

**Comment by thesergeev** [comment]
Hello!
I found solutions. But don't know correct it or not. 
`function perf_sroll_bar() {
	if ($(window).width() > 767) {
		$('.page_content').perfectScrollbar();
		$('.page_content').perfectScrollbar('update');
	} else {
		$('.page_content').perfectScrollbar('destroy');	
	}
};
perf_sroll_bar();
$(window).resize(function() {perf_sroll_bar});`

**Comment by DanielApt** [comment]
@merzikain By the sound of it, you are successfully destroying the scrollbar, but not re-creating it properly. To create the scrollbar you have to call `$('.perscroll').perfectScrollbar({suppressScrollX:true})`.

@thesergeev Is using the right approach, however we can double check wether we need to `update` or `initialise` perfect scrollbar.

You can see how I'm doing that here https://jsfiddle.net/gbfLazpx/12/

If you need any further help, feel free to re-open this issue.

**Comment by merzikain** [comment]
Thing is, I did it similar to what @thesergeev posted before posting my issue. The difference being that instead of using 'update' I used {suppressScrollX:true}. But when I scaled the window down to trigger the destroy, it would destroy it, but when I scaled the window back up it would not recreate it.

The difference between what I tried and what you provided in the jsfiddle is that I did not set my container as a variable and I did not check to see if .ps-container was already set. I also did not put my code in a new function (I have another script that auto triggers a window resize a moment after the page loads for another purpose).

The biggest difference, and the actual problem, is that I have a global variable called "ww" that contains the window width. Another resize function controls it but it seems that it doesn't change before the perfectScrollbar checking happens. When I replace ww with $(window).width() it works fine.

Thanks.
